src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,171 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.26.0.dev1 (02/24/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Upgrade to Pex 2.1.4 (#8881)
+  `PR #8881 <https://github.com/pantsbuild/pants/pull/8881>`_
+
+* Register `goal2` as `goal` if in v2-exclusive mode. (#9164)
+  `PR #9164 <https://github.com/pantsbuild/pants/pull/9164>`_
+
+* Turn on `--v2` by default now that the V2 backends are not loaded by default (#9007)
+  `PR #9007 <https://github.com/pantsbuild/pants/pull/9007>`_
+
+* Reapply "Switch to dot idea project type" PR (#9105)
+  `PR #9105 <https://github.com/pantsbuild/pants/pull/9105>`_
+
+* [export-dep-as-jar] Include transitive target dependencies in export-dep-as-jar (#9146)
+  `PR #9146 <https://github.com/pantsbuild/pants/pull/9146>`_
+
+* Remove deprecated passthrough args for `login`, `lint.mypy`, and `fmt.isort` (#9134)
+  `PR #9134 <https://github.com/pantsbuild/pants/pull/9134>`_
+
+* Remove pytest passthrough args deprecation. (#9126)
+  `PR #9126 <https://github.com/pantsbuild/pants/pull/9126>`_
+
+* Remove deprecated `--requirements` options in favor of `--version` and `--extra-requirements` (#9118)
+  `PR #9118 <https://github.com/pantsbuild/pants/pull/9118>`_
+
+* Remove deprecated `pantsbuild.pants.testinfra` wheel (#9117)
+  `PR #9117 <https://github.com/pantsbuild/pants/pull/9117>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Respect precise file arguments with V2 `./pants test` (#9120)
+  `PR #9120 <https://github.com/pantsbuild/pants/pull/9120>`_
+
+* Add support for precise file arguments to `fmt2` and `lint2` (#9133)
+  `PR #9133 <https://github.com/pantsbuild/pants/pull/9133>`_
+
+* v2 repl goal for python (#9077)
+  `PR #9077 <https://github.com/pantsbuild/pants/pull/9077>`_
+
+* Add Python `docformatter` auto-formatter (#9139)
+  `PR #9139 <https://github.com/pantsbuild/pants/pull/9139>`_
+
+* Add Pylint as a V2 linter (#8979)
+  `PR #8979 <https://github.com/pantsbuild/pants/pull/8979>`_
+
+* Allow disabling V1 isort by deregistering `pants.backend.python.lint.isort` (#9140)
+  `PR #9140 <https://github.com/pantsbuild/pants/pull/9140>`_
+
+* [jvm] Introduce runtime_platform; bump export version to 1.1 (#9003)
+  `PR #9003 <https://github.com/pantsbuild/pants/pull/9003>`_
+
+* Fetch sources and javadoc jars for 3rd party deps when exporting with 'export-dep-as-jar' (#9168)
+  `PR #9168 <https://github.com/pantsbuild/pants/pull/9168>`_
```

Comment:
Henry recommended putting this as new feature but Yi thought bug fix. It sounds like it could be either. I put it as new feature because it introduces new flags for the user and I want to optimize for them discovering that they can now use new flags.

cc @hrfuller @wisechengyi